lib/public/views/LhcFills/Overview/LhcFillsOverviewModel.js

```diff
@@ -75,10 +82,53 @@ export class LhcFillsOverviewModel extends OverviewPageModel {
      *
      * @return {boolean} true if the stable beams filter is active
      */
-    isStableBeamsOnly() {
+    getStableBeamsOnly() {
```

Comment:
In general in programming, for boolean values, we do not use `get` but instead something that suggest the answer it is a yes/no. Thus, methods in this scenarios are using `is...` or `has...`
This change will be reverted